Makefile

```diff
@@ -37,6 +37,7 @@ EXPORTS := \
   CRYSTAL_CONFIG_LIBRARY_PATH="$(shell crystal env CRYSTAL_LIBRARY_PATH)" \
   CRYSTAL_CONFIG_BUILD_COMMIT="$(CRYSTAL_CONFIG_BUILD_COMMIT)" \
 	SOURCE_DATE_EPOCH="$(SOURCE_DATE_EPOCH)"
+SPEC_EXPORTS := CRYSTAL_LIBRARY_PATH="$(shell crystal env CRYSTAL_LIBRARY_PATH)"
```

Comment:
Any second thought on https://github.com/crystal-lang/crystal/pull/9423/files#r435373542 ?

Reply:
Ah, no. I just copy pasted here because I was counting on having to rebase on #9423 anyways :)

Reply:
While doing #9423 I consider adding $(EXPORTS) to $(O)/all_spec, $(O)/compiler_spec.

CRYSTAL_LIBRARY_PATH should not be needed in std_spec (as long as we don't build the compiler) and the installed compiler in PATH or .build/crystal has that set.

Would that work? I can add those changes in #9423

Reply:
I considered this too, but wasn't too sure because it sets so many. But I guess it does no harm :)